reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md

```diff
@@ -0,0 +1,133 @@
+|* create virtual server instance to host basic management services like DNS, NTP, NFS|Management services VPC instance|CreateLinux VPC instance that may host management components. Preconfigure ACL and security group rules to allow communication for basic management componets (NFS - ports XXX, NTP - ports XXX, DNS - ports XXX)|Modify number of virtual server instances and allowed ports in preset or perform the modifications manually|
+|* ensure financial services compliancy for VPC services  \n * perform network setup of all created services  \n * perform network isolation of all created services  \n * ensure all created services are inteconnected with each other|Secure landing zone components|Create a minimum set of required components for a secure landing zone|Create a modified set of required components for a secure landing zone in preset|
+{: caption="Table 1. VPC architecture decisions" caption-side="bottom"}
+
+### PowerVS workspace architecture decisions
+{: #pvs-components}
+
+| Requirement | Component | Choice | Alternative choice |
+|-------------|-----------|--------------------|--------------------|
+|* connect PowerVS workspace with VPC services|Cloud connections|Setup two redundant cloud connections|                    |
+|* configure network for management of all the instances  \n * troughput and latency are not relevant|Management network|Configure private network with default configurations and attch it to both cloud connections|                    |
+|* configure separate network for backup purposes with higher data throughput|Backup network|Configure separate private network with default configurations and attch it to both cloud connections. Networks characteristics might be adapted by the users manually (e.g., to improve throughput)|                    |
+|* preload OS images relevant for customer workload|Pre-loaded OS images|Preload Linux OS images for SAP workload. Keep the number of preloaded images at minimum to save corsts.|Modify the input parameter that specify the list of preloaded OS images.|
+|* preload public SSH key that will be injected into every OS deployment|Pre-loaded SSH public key|Preload customer specified SSH public key|                    |
+{: caption="Table 2. PowerVS workspace architecture decisions" caption-side="bottom"}
+
+### PowerVS management services architecture decisions
+{: #vpc-components}
+
+| Requirement | Component | Choice | Alternative choice |
+|-------------|-----------|--------------------|--------------------|
+|* ensure public internet connectivity from all the instances to be deployed in PowerVS workspace|SQUID proxy|Setup SQUID proxy software on Linux virtual server instance runnning in edge VPC|                    |
```

Comment:
Can you make the same changes in the following tables?
- initial capitalization
- replace e.g., with For example
- fix typos (`runnning`, `Sinchronize` and `sinchronisation`)